test/parallel/test-httpsServerOptions.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const https = require('https');
+const fs = require('fs');
+
+var options = {
+  key: fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem', 'ascii'),
+  crt: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem', 'ascii')
+};
+
+assert.throws(function() {
+  https.createServer(options, function (req, res) {
```

Comment:
For this usecase, the callback passed should not be called. So, its better to use `assert.fail` there.